roles/provision_vm/README.md

```diff
@@ -406,7 +406,7 @@ N/A
     Set the guest ID.
     This parameter is case sensitive.
     This field is required when creating a virtual machine, not required when creating from the template.
-    Valid values are referenced here: https://code.vmware.com/apis/358/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+    Valid values are referenced here: https://vdc-download.vmware.com/vmwb-repository/dcr-public/184bb3ba-6fa8-4574-a767-d0c96e2a38f4/ba9422ef-405c-47dd-8553-e11b619185b2/SDK/vsphere-ws/docs/ReferenceGuide/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
```

Comment:
As the link is quite long, maybe adding hyperlink like [here](https:...) would be better

Reply:
Fixed it thanks.